### DISCUSSION TITLE
Is there a way to get more information about the results of each file in the NZB?

### DISCUSSION THREAD

**Discussion by Thraka** [sabnzbd/sabnzbd#2971] [General]
While the NZB is downloading, you can see a list of each file and if it was able to get the blocks and create the file. But if some files fail, failing the entire download, the summary doesn't list all of the details. There may be some good files in the collection and some corrupted files, but I can't tell.

**Comment by thezoggy** [comment]
not sure what the intent of you opening this discussion here rather than asking in our support places (forum/discord/irc)

you dont really know whats good or bad until you verify/repair. which you cant really do until the job is done. which you can see results of par2 if you do debug logs.

**Reply by Thraka** [reply]
I have an account here 😁 and discussions are turned on. I figured that if this feature didn't exist, I would then file a feature request here too.

It does a verify/repair and fails for whatever reason. What files did it need to repair? I hate having to jump down to command line tools and run manual par2 or checksum checks when sabnzbd has already done that work but doesn't seem to surface that data.

**Reply by thezoggy** [reply]
you can just look at your sab debug logs, but generically speaking if it cant be repaired it doesnt really matter ? as most likely people dont want a partial output of the job.. youd just care that it failed and would move along to trying another nzb
